CONTRIBUTOR_ROLES.md

```diff
@@ -115,9 +115,13 @@ Maintainers will vote publicly on the issue, expressing their support via a GitH
 
 After a [decision has been made](https://github.com/instructlab/community/blob/main/governance.md#decision-making-at-the-instructlab-org-level), a Maintainer will create a PR to add you in the [MAINTAINER file](https://github.com/instructlab/community/blob/main/MAINTAINERS.md) within three weeks.
 
+All Maintainers are Triagers. If a candidate is not a Triager at the time of the nomination, they should be added to both Triagers and Maintainers teams.
```

Comment:
Is this saying that you can nominate a user to be a maintainer without them serving as a triager? Does this circumvent the [contributor ladder](https://github.com/instructlab/community/blob/main/CONTRIBUTOR_ROLES.md#current-roles)?